salt/modules/rh_service.py

```diff
@@ -470,22 +472,34 @@ def reload_(name):
 
 def status(name, sig=None):
     '''
-    Return the status for a service, returns a bool whether the service is
-    running.
+    Returns a bool indicating whether the service is running.
+
+    If ``name`` contains globbing characters (such as ``*``, ``?``, or anything between brackets),
+    returns a dict name/bool instead of a simple bool.
```

Comment:
The wording needs to be improved here. Instead of "dict name/bool", it should be "dict mapping service names to True/False values".